polaris-react/src/components/Tile/Tile.stories.tsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import type {ComponentMeta} from '@storybook/react';
+import {Badge, Box, Tile, Text} from '@shopify/polaris';
+
+export default {
+  component: Tile,
+} as ComponentMeta<typeof Tile>;
+
+export function Default() {
+  return (
+    <Tile spacing="1" columns="2">
+      <Box background="surface" borderRadius="2" padding="5" shadow="card">
```

Comment:
It doesn't seem like `Box` is rendering as expected in the stories? I'm not seeing any of the styles like `padding` or `border-radius` rendered. Maybe for now it makes sense to use a `div` instead so the focus is on `Tile`.

Reply:
I'm seeing the correct border radius on the chromatic story that's created from the PR https://5d559397bae39100201eedc1-hozvnhnshv.chromatic.com/?path=/story/all-components-tile--default